gitgud/operations.py

```diff
@@ -155,6 +157,54 @@ def commit(self, commit_message, parents, time_offset):
                 skip_hooks=True)
         return commit_obj
 
+    def file_in_commit(self, commit, filepath):
+        commit = self.repo.commit(commit)
+        return filepath in commit.tree
+
+    def get_commit_content(self, commit, filepath):
+        if commit in self._streamed_content:
+            return self._streamed_content[commit]
```

Comment:
This doesn't make sense, are we reading from the file or the commit? This dict should need `filepath`